infrastructure/api-configuration/nginx_config.conf

```diff
@@ -60,6 +60,34 @@ http {
             proxy_set_header Host $http_host;
             proxy_set_header X-Cluster-Client-Ip $remote_addr;
             proxy_pass http://app;
+
+            if ($request_method = 'OPTIONS') {
```

Comment:
I am surprised to see this here. Normally CORS is handled at the application layer, and we already have CORS middleware installed for the API.

I'd rather this stuff not be site wide in case we want to have parts of the site un-CORS (admin pages, internal tools and dashboards, etc).

I was expecting this to be a plain http -> https rewrite - more generally, can you explain what's actually happening in this PR?

I see that this includes both Let'sEncrypt's CertBot and AWS's Certificate Manager - which service is generating the SSL certificate? How is it being verified? How is it being renewed? Where is this process documented?